dialog/base.go

```diff
@@ -22,10 +22,11 @@ const (
 type Dialog interface {
 	Show()
 	Hide()
-	SetDismissText(label string)
-	SetOnClosed(closed func())
+	SetDismissText(string)
+	SetOnClosed(func())
+	SetTitle(string)
 	Refresh()
-	Resize(size fyne.Size)
+	Resize(fyne.Size)
```

Comment:
I don't know to what extent people create truly custom dialogs using the dialog interface (if it is even possible), but isn't this a breaking change?

Reply:
Yes I guess that is strictly true. I suppose I need another approach.
The alternative is to expose new concrete types I suppose (better than many new `MutableDialog` interfaces).